packages/web/src/styles/Button.js

```diff
@@ -12,36 +12,42 @@ const filters = ({ colors: { borderColor } }) => css`
 		padding: 5px 8px;
 		font-size: 0.85rem;
 		position: relative;
+	}
+`;
 
-		span:first-of-type {
-			max-width: 260px;
-			white-space: nowrap;
-			overflow: hidden;
-			text-overflow: ellipsis;
-			margin-right: 26px;
-		}
-
-		span:last-of-type {
-			display: flex;
-			height: 100%;
-			top: 0;
-			right: 8px;
-			position: absolute;
-			align-items: center;
-			border-left: 1px solid ${borderColor || '#fff'};
-			padding-left: 8px;
-			margin-left: 8px;
-		}
-
-		&:hover,
-		&:focus {
-			span:first-of-type {
-				text-decoration: line-through;
-			}
-		}
+Filter.Value = styled.span`
+	max-width: 200px;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	flex-wrap: nowrap;
+	flex-wrap: nowrap;
+	width: max-content;
+
+	&:hover,
+	&:focus {
+		text-decoration: line-through;
 	}
 `;
 
+
+Filter.ImageValue = styled(Filter.Value)`
+	display: flex;
+	justify-content: center;
+	align-items: center;
+`;
+
+Filter.CloseIcon = styled.span`
+	display: flex;
+	height: 100%;
+	top: 0;
+	right: 8px;
+	align-items: center;
+	border-left: 1px solid ${props => props.theme.colors.textColor || '#000'};
```

Comment:
In this case, it is good if it matches the text color. See below image. If we had black color, which matched the text color, instead of white color, then it would be much better.
![image](https://github.com/appbaseio/reactivesearch/assets/52590791/19232b93-f551-4bae-beac-080ca835c176)

Reply:
@SavvyShah Makes sense, Please share a demo of the same for the dark theme, we want to make sure the dark theme works well with this change.

Reply:
In dark theme, component would look like this
<img width="376" alt="image" src="https://github.com/appbaseio/reactivesearch/assets/52590791/a470fb50-c3c5-4b98-8f77-3934ba5594f2">